### PR TITLE
include hash of prior block in each bundle

### PR DESCRIPTION
**Pull request by darinmcgill**

### CODE REVIEW

File: javascript/implementation/Bundler.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export class Bundler implements BundleView {
      * @param bundleInfo the bundle metadata to add when serializing
      * @returns serialized
      */
-    seal(bundleInfo: BundleInfo, keyPair: KeyPair): void {
+    seal(bundleInfo: BundleInfo, keyPair: KeyPair, prior_hash?: Bytes): void {
```
Comment (char26) [inline javascript/implementation/Bundler.ts:92]: What is the reason for using snake case here?

File: javascript/implementation/IndexedDbStore.ts [hunk 2]
```diff
@@ -500,13 +502,13 @@ export class IndexedDbStore implements Store {
                     clearanceId: changeAddressTuple,
                     purging: clearanceBuilder.getPurge()
                 };
-                await wrappedTransaction.objectStore("clearances").add(clearance);
+                await trxn.objectStore("clearances").add(clearance);
                 continue;
             }
             throw new Error("don't know how to apply this kind of change");
         }
         // console.log(`finished addBundleHelper for: ` + JSON.stringify(bundleInfo));
```
Comment (char26) [inline javascript/implementation/IndexedDbStore.ts:510]: Remove this comment now?

File: proto/log_file.proto [hunk 3]
```diff
@@ -10,15 +11,20 @@ import "proto/claim.proto";
     structured.
 */
 message LogFile {
+
+
+    fixed32 magic_number = 1;  // should be 1263421767
+
```
Comment (char26) [inline proto/log_file.proto:17]: Spacing and comment supposed to be there?

**Review by char26 — APPROVED** [review]

File: javascript/implementation/Bundler.ts (hunk 1 above)
Comment (darinmcgill) [reply javascript/implementation/Bundler.ts:92]: Too much context switching between python and Javascript/Typescript.

File: javascript/implementation/IndexedDbStore.ts (hunk 2 above)
Comment (darinmcgill) [reply javascript/implementation/IndexedDbStore.ts:510]: yeup

File: proto/log_file.proto (hunk 3 above)
Comment (darinmcgill) [reply proto/log_file.proto:17]: Adding another comment to clarify.

**Pull request merged**